x-pack/plugins/security_solution/cypress/e2e/cases/attach_alert_to_case.cy.ts

```diff
@@ -41,8 +41,9 @@ describe('Alerts timeline', () => {
     });
 
     it('should not allow user with read only privileges to attach alerts to cases', () => {
-      // Disabled actions for read only users are hidden, so actions button should not show
-      cy.get(TIMELINE_CONTEXT_MENU_BTN).should('not.exist');
+      // Disabled actions for read only users are hidden, so only open alert details button should show
+      expandFirstAlertActions();
+      cy.get(ATTACH_ALERT_TO_CASE_BUTTON).should('not.exist');
```

Comment:
An alert can be attached to a case by selecting `Add to existing case` or `Add to new case`, per the `data-test-subj` looks like we are checking just the to the first scenario, it would be nice to add another assertion to make sure the `Add to new case` option is not displayed as well.

Reply:
Yea, I just updated the existing test since the open details page is visible. But I went ahead and added the other test here as well 👍🏾

Reply:
Awesome!!

NIT: In order to improve execution time, in this specific case you can have the test as `should not allow user with read only privileges to attach alerts to a case` and have the 2 assertions on the same test.